### PR TITLE
<SignatureInput> - Add a11y support

### PR DESCRIPTION
**Pull request by giladsegal**

### CODE REVIEW

File: packages/wix-ui-core/src/components/signature-input/dataHooks.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const DataHooks = {
+  a11yInput: 'signature-input--a11y-input',
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/components/signature-input/dataHooks.ts]: Double `--` intended?

File: packages/wix-ui-core/src/components/signature-input/constants.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
-export const SIGNNATURE_INPUT_METADATA = {
-  displayName: 'SignnatureInput',
+export const SIGNATURE_INPUT_METADATA = {
+  displayName: 'SignatureInput',
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/components/signature-input/constants.ts]: Curer of OCDs

File: packages/wix-ui-core/src/components/signature-input/signing-pad/SigningPad.tsx [hunk 3]
```diff
@@ -138,45 +204,77 @@ class SigningPadComp extends React.Component<SigningPadProps> {
...
+    this.setState(updateA11yValue(e.target.value));
+    const ctx = this.canvasEl.getContext('2d');
+    ctx.font = '18px Sacramento';
+    ctx.fillText(e.target.value, 5, this.canvasEl.offsetHeight / 2);
+    this.invokeIfDefined(onDraw)(e);
+  };
+
+  isA11yInputEmpty = () => !this.state.a11yInputValue;
+
+  invokeIfDefined = <T extends unknown>(fn?: (e: T) => void) => (e: T) => {
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/components/signature-input/signing-pad/SigningPad.tsx]: I like this, it should be a util some day :D

File: packages/wix-ui-core/src/components/signature-input/dataHooks.ts (hunk 1 above)
Comment (giladsegal) [reply packages/wix-ui-core/src/components/signature-input/dataHooks.ts]: yeah I like the BEM style `component--part`

**Pull request merged**